packages/backend/package.json

```diff
@@ -6,7 +6,7 @@
   "types": "./index.d.ts",
   "scripts": {
     "build": "tsc -p tsconfig.build.json",
-    "prebuild": "yarn clean",
+    "build:dependencies": "cd ../.. && yarn build:dependencies && cd -",
```

Comment:
There is no need to go back in this script. I think the working directory stays the same outside.